boot/zephyr/serial_adapter.c

```diff
@@ -148,7 +148,13 @@ boot_uart_fifo_callback(struct device *dev)
 			cmd->line[cur++] = byte;
 		}
 
-		if (byte ==  '\n') {
+#ifdef CONFIG_BOOT_SERIAL_UART_DETECT
+		if (byte == '\r') {
+			byte = '\n';
+		}
+#endif
+
+		if (byte ==  '\n'){
```

Comment:
Missed a space between "){".

Reply:
Thanks,  I fixed . Is there a tool or script to detect this format?

Reply:
It might be possible to adapt checkpatch.pl from the Linux kernel, but it probably also detects a lot of things we do want. It might also be possible to configure indent or equivalent to check for this.